.github/workflows/build.yml

```diff
@@ -85,6 +85,13 @@ jobs:
     needs: build
     env:
       SHA_TAG: ${{needs.build.outputs.sha_tag}}
+      TF_VAR_paas_settings__basic_auth__password: ${{ secrets.TF_VAR_PAAS_SETTINGS__BASIC_AUTH__PASSWORD }}
+      TF_VAR_paas_settings__dttp__client_secret:  ${{ secrets.TF_VAR_PAAS_SETTINGS__DTTP__CLIENT_SECRET }}
```

Comment:
Can we organise this one to sit along the rest of `TF_VAR_paas_settings__dttp*`?

Reply:
done already..